chain/stmgr/forks.go

```diff
@@ -208,7 +211,21 @@ func (sm *StateManager) handleStateForks(ctx context.Context, root cid.Cid, heig
 	return retCid, nil
 }
 
-func (sm *StateManager) hasExpensiveFork(ctx context.Context, height abi.ChainEpoch) bool {
+// Returns true if executing the current tipset would trigger an expensive fork.
+//
+// - If the tipset is the genesis, this function always returns false.
+// - If inclusive is true, this function will also return true if applying a message on-top-of the
```

Comment:
This comment looks out of date with the current solution, `inclusive` is no longer a parameter and there are now two functions which should probably each have a comment.

Reply:
Ah, yes. Let me fix that.